### PR TITLE
BCNM content, step 1: fill in missing bcnm_battlefield records

### PR DESCRIPTION
**Pull request by wrenffxi**

### CODE REVIEW

**Comment by teschnei** [comment]
Hmm.. Do we want to comment out the ones that aren't implemented?  That makes it easy to see at a glance which ones aren't (rather than having to check each zone's folder)

Not sure if that would be helpful or not to people looking to implement BCNMs

**Comment by wrenffxi** [comment]
I can do that if you want, though it's already sort of covered by the bcnm.lua in bcnmcrap
https://github.com/DarkstarProject/darkstar/blob/bcnmcrap/scripts/globals/bcnm.lua

**Comment by teschnei** [comment]
If there's already a central place to see it, then yeah don't bother here.
Two places to remember to uncomment isnt helping more than one

File: sql/bcnm_battlefield.sql [hunk 1]
```diff
@@ -1821,25 +2201,25 @@ INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941084,3);
...
-INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941100,3);  -- Gosspix_Blabberlips				
+INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941100,3);  -- Gosspix_Blabberlips              
 INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941381,3); 
 INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941382,3); 
-INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941383,3); 						
+INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941383,3);                      
 INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941395,3); 
 INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941396,3); 
-INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941397,3); 					
+INSERT INTO `bcnm_battlefield` VALUES (1287,1,16941397,3);
```
Comment (TeoTwawki) [inline sql/bcnm_battlefield.sql]: pls trim trailing whitespace

Comment (wrenffxi) [reply sql/bcnm_battlefield.sql]: Done.

**Pull request merged**